integration-cli/docker_cli_external_volume_driver_unix_test.go

```diff
@@ -279,6 +279,24 @@ func (s *DockerExternalVolumeSuite) TearDownSuite(c *check.C) {
 	c.Assert(err, checker.IsNil)
 }
 
+func (s *DockerExternalVolumeSuite) TestVolumeCLICreateOptionConflict(c *check.C) {
+	dockerCmd(c, "volume", "create", "test")
+
+	newVolumePlugin(c, "otherdriver")
```

Comment:
`volumePlugin` runs a `httpServer` thread that needs to be closed at the end of this test.